airflow/providers/influxdb/hooks/influxdb.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, conn_id: str = default_conn_name, *args, **kwargs) -> None:
         self.influxdb_conn_id = conn_id
         self.connection = kwargs.pop("connection", None)
         self.client = None
-        self.extras = None
+        self.extras: Dict = {}
```

Comment:
Does `extras` need to be an instance attribute of the hook? Looks like `self.extras` is only accessed and used in the `get_conn()` method. Could get rid of the typing issues that way.

Might be the same behavior for some of the other hook attributes as well that are initialized to `None`.

Reply:
It doesnt, will remove, I thought it was used in the BaseHook, looks like its not.